frontend/src/static/js/v_summary.js

```diff
@@ -273,22 +273,18 @@ window.vSummary = {
       }
 
       let minDate = window.app.sinceDate;
-      let maxDate = '';
       this.filtered.forEach((repo) => {
         repo.forEach((user) => {
           const { commits } = user;
           if (commits.length) {
             const date1 = commits[0].date;
-            const date2 = commits[commits.length - 1].date;
             if (!minDate || minDate > date1) {
               minDate = date1;
```

Comment:
think this entire chunk can be removed as well?

Reply:
@eugenepeh cannot because the commits[0] date for each user in each repo varies. `window.app.sinceDate` is also not always defined.

Reply:
I see 👍